web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -30,6 +30,11 @@ This PDEP proposes that:
 This will bring **immediate benefits to users**, as well as opening up the door for significant further
 benefits in the future.
 
+**NOTE**: while this PDEP mentions adding pyarrow as a required dependency in
+pyarrow 3.0, this aspect has been delayed until after pandas 3.0. Therefore,
+pandas 3.0 will not have a hard requirement on pyarrow and only use it by
+default (for strings) when installed.
```

Comment:
Maybe reference the appropriate part of PDEP14? and/or PDEP15?

Reply:
Added a link to PDEP-14 and rephrased the last sentence slightly